components/table/README.md

```diff
@@ -228,7 +232,127 @@ When tabular data can be sorted, the `<d2l-table-col-sort-button>` can be used t
 | `desc` | boolean | Whether sort direction is descending | false |
 | `nosort` | boolean | Column is not currently sorted. Hides the ascending/descending sort icon. | false |
 | `position` | string | Position of the button content. Accepted values are 'start', 'center', and 'end'. | 'start' |
-| `source-type` | string | The type of data in the column. Used to set the title. Accepted values are 'words', 'numbers', and 'dates'. | 'unknown' |
+| `source-type` | string | The type of data in the column. Used to set the title. Accepted values are 'words', 'numbers', and 'dates'. This is only applicable to cases that are not using the multi-faceted dropdown. | 'unknown' |
+
+### Slots
+| Name | Description |
+|---|---|
+| `Default` | Column header text |
+| `items` | Multi-facted sort items. Generally assigned to the `slot` attribute on a nested `d2l-table-col-sort-button-item`. |
+
+### Slotted Item [d2l-table-col-sort-button-item]
+
+This is a radio menu item to be used within the `d2l-table-col-sort-button` component for a [multi-faceted sort](#multi-faceted-sort-button).
+
+<!-- docs: demo code properties autoSize:false align:start name:d2l-table-col-sort-button-item -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/table/table-col-sort-button.js';
+  import '@brightspace-ui/core/components/table/table-col-sort-button-item.js';
+</script>
+<d2l-table-col-sort-button style="--d2l-table-cell-padding: 15px;" desc>
```

Comment:
Is the custom padding needed here?

Reply:
It is, otherwise there is no padding on the col-sort-button component. Thoughts on if the component should have a default fallback as well?

Reply:
I don't think I fully understand the padding usage scenarios. My hope would be that they can just use `d2l-table-col-sort-button` out of the box. If they have to provide their own padding, are we setting ourselves up for inconsistency or maintenance headaches down the road?

Reply:
It's currently just dependant on `--d2l-table-cell-padding` as there is the assumption that this is used within a table that is using the table styles. It can be used out of the box for a regular consumer who puts it in a table, I just didn't do that in this example.